lg_common/AdhocBrowser[] browsers
string scene_slug
bool sync_windows
